### ISSUE TITLE
How to tune hyper-parameters?

### ISSUE THREAD

**Issue by super233** [lhoyer/DAFormer#41]
It's really an awesome transformer based UDA work, thanks for your sharing codes.

I want to ask an open question about UDA.

Considering that the label of target domain is not available, it's impossible to directly evaluate model performance based on the target domain, then how to tune hyper-parameters? And I mean that the validation set of target domain can not be used for tuning hyper-parameters.

Looking forward your reply. :-)
